plugins/ldap/girder_ldap/__init__.py

```diff
@@ -91,6 +91,9 @@ def _ldapAuth(event):
             conn.bind_s(server['bindName'], server['password'], ldap.AUTH_SIMPLE)
 
             searchStr = '%s=%s' % (server['searchField'], login)
+            if server['queryFilter']:
```

Comment:
Sorry, one more change needed here. This will fail on existing LDAP settings that don't have the `queryFilter` key in them. So in order not to break existing users, let's change this to `if server.get('queryFilter'):`

Reply:
I see, I thought that my change in settings.py added a default but that was just the validation, thanks for the hint, I'll fix it.